assets/js/blocks/cart-checkout-shared/sidebar-notices/index.tsx

```diff
@@ -66,12 +64,19 @@ const withSidebarNotices = createHigherOrderComponent(
 			<>
 				{ ( isCart || isCheckout ) && (
 					<InspectorControls>
+						{ isWcVersion( '6.9.0', '>=' ) ? (
+							<DefaultNotice
+								block={ isCheckout ? 'checkout' : 'cart' }
+							/>
+						) : (
+							<LegacyNotice
+								block={ isCheckout ? 'checkout' : 'cart' }
+							/>
+						) }
+
```

Comment:
This logic should handle older versions which doesn't support the setting needed.